src/styles/card.css

```diff
@@ -0,0 +1,94 @@
+.card {
+  display: inline-flex;
+  flex-direction: column;
+  max-width: 198px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  align-self: center;
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+}
+
+.card__text {
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.card__title {
+  font-size: 12px;
+  font-weight: 500;
+  width: 166px;
+  height: 36px;
+  line-height: 18px;
+  margin-bottom: 4px;
+}
+
+.card__product-code {
+  height: 14px;
+  line-height: 14px;
+  width: 98px;
+  font-weight: 400;
+  font-size: 10px;
+  color: var(--color-secondary);
+  margin-bottom: 16px;
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.card__reviews {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 16px;
+  color: var(--color-main);
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+  color: var(--color-secondary);
+}
+
+.card__price-text {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card__price-value {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--color-main);
+}
+
+.card__buttom {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 40px;
+  border-radius: 5px;
+  border: 1px solid var(--color-blue) transparent;
+  background-color: var(--color-blue);
+
+  color: var(--color-white);
+  font-weight: 700;
+  font-size: 14px;
+  text-decoration: none;
+
+  transition: 0.4s;
```

Comment:
specify the properties for which you need to use transition (it sets all by default, which will affect in bad way on website performance)